src/main/java/uk/gov/hmcts/reform/demo/config/AzureConfigurationClient.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.demo.config;
```

Comment:
package needs renaming from demo to pip.account.management

Reply:
Good spot - I've renamed this now